plugins/org.eclipse.osee.ats.api/src/org/eclipse/osee/ats/api/config/AtsConfigEndpointApi.java

```diff
@@ -137,6 +138,12 @@ public interface AtsConfigEndpointApi {
    @Produces(MediaType.APPLICATION_JSON)
    public Version getVersion(@PathParam("verId") ArtifactId verId);
 
+   @POST
+   @SkipAtsConfigJsonWriter
```

Comment:
Remove the "/create".  That is assumed by POST.  The method can still be createVersion.

Reply:
I thought of that but in some cases we are explicit.  I'll update it.